os/arch/arm/src/common/up_allocateheap.c

```diff
@@ -181,8 +200,12 @@ void up_add_kregion(void)
 		}
 
 		heap_size = kregionx_end - heap_start;
-
-		lldbg("start = 0x%x size = %d\n", heap_start, heap_size);
+#ifdef CONFIG_SECURITY_LEVEL
+		if (get_security_level() == LOW_SECURITY_LEVEL)
+#endif
```

Comment:
How about we implement a new macro and use it for all secure dbg logs? We can avoid adding the `if` statements in many files.